app/poros/food.rb

```diff
@@ -0,0 +1,12 @@
+class Food
+  attr_reader :total_results, :id, :description, :brand_owner, :ingredients
+
+  def initialize(data)
+    @id = data[:fdcId]
+    @description = data[:description]
+    @brand_owner = data[:brand_owner]
+    @ingredients = data[:foodNutrients].map do |nutrient|
+      nutrient[:nutrientName]
+    end
```

Comment:
It doesn't feel great to do a `map` within the initialize. Could this live in a method?
```
def initialize(data)
  ...
  @ingredients = ingredient_names(data[:foodNutrients])
end

def ingredient_names(nutrients)
  nutrients.map do |nutrient|
    nutrient[:nutrientName]
  end
end